src/transaction.js

```diff
@@ -51,8 +42,9 @@ function mkNullDataReplayScript (
   )
 }
 
-/* More info: https://github.com/ZencashOfficial/zen/blob/master/src/script/standard.cpp#L377
+/*
```

Comment:
This change is for aligning with the style of comments in other parts of this files.